src/mte_tag.adoc

```diff
@@ -295,6 +295,31 @@ local to a function. These accesses are deemed to be safe and thus are not
 subject to tag checks.
 =====
 
+==== Explicit tag check
+Zimt defines an explicit tag check instruction taken from `MOP.RR.1` of zimop
+extension, `checktag rs1, #chunk_count` which extracts `pointer_tag` from `rs1`
+and performs load tag operation for memory chunks (count = `#chunk_count`)
+starting at virtual address `rs1`. If any of the loaded `mc_tag` value(s)
+mismatches with the `pointer_tag` specified in `rs1`, then software check
+exception is raised with tval = 4. If memory tagging is enabled in the
+execution environment then `checktag` instruction always makes the check
```

Comment:
This is an interesting consideration. Can you give your rational for ignoring the PTE.MTAG of the *data*?

Reply:
Take a look now.

Reply:
@fmayer 
looks okay to you?